remappings.txt

```diff
@@ -3,3 +3,5 @@ ds-test/=lib/forge-std/lib/ds-test/src/
 erc4626-tests/=lib/openzeppelin-contracts/lib/erc4626-tests/
 forge-std/=lib/forge-std/src/
 openzeppelin-contracts/=lib/openzeppelin-contracts/
+@openzeppelin/contracts/=lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/
```

Comment:
Pls make sure there is 1 reference to the standard OZ lib and 1 ref to the upgradeable one. Right now there are 3 refs to the standard OZ lib and 1 ref to the upgradeable one [here](https://github.com/ubiquity/nft-rewards/blob/3a87e51b18c82730b96459a3eff5149e1c7122f9/remappings.txt).